src/components/resource/resource-show.js

```diff
@@ -213,6 +214,9 @@ class ResourceShow extends Component {
       sections
     } = this.state;
 
+    const userDefinedFields = getUserDefinedFields(model);
+    const isCustomLabelsViewed = model.isTitleCustom || (model.isSelected && !model.isTitleCustom);
```

Comment:
1) Please rename it to showCustomLables or similar as as viewed means that someone already saw it
2) I believe model.isTitleCustom || (model.isSelected && !model.isTitleCustom) could be rewritten to 
model.isTitleCustom || model.isSelected